src/audiovideocontroller/AudioVideoControllerFacade.ts

```diff
@@ -2,11 +2,16 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import AudioVideoObserver from '../audiovideoobserver/AudioVideoObserver';
+import VideoSendingAttendee from '../videostreamindex/VideoSendingAttendee';
 
 export default interface AudioVideoControllerFacade {
   addObserver(observer: AudioVideoObserver): void;
   removeObserver(observer: AudioVideoObserver): void;
   start(): void;
   stop(): void;
   getRTCPeerConnectionStats(selector?: MediaStreamTrack): Promise<RTCStatsReport>;
+  /**
+   * Gets available remote attendeeIds with their externalUserIds who have turned their video on.
+   */
+  getRemoteVideosAvailable(): VideoSendingAttendee[];
```

Comment:
Suggest: `getAvailableRemoteVideoAttendees`

Reply:
Ok, Thanks, and can you also please review / suggest for observer method name: `remoteVideosAvailableDidChange`?